assets/js/components/notifications/BannerNotification.js

```diff
@@ -159,6 +165,25 @@ function BannerNotification( {
 		}
 	}
 
+	const handleAnchorLinkClick = ( e, link ) => {
+		e.preventDefault();
+
+		if ( isHashOnly( link ) ) {
```

Comment:
I feel that this check is currently not required as the only links passed to the `<Link>` component below are hash anchors (in `SetupSuccessBannerNotification.js`. However, I have added it as I think that is what the IB suggests but correct me if I misunderstood the IB.

Reply:
Yes, let's keep this check, but rework this function so that it prevents the default behaviour only when the link is hash and does nothing otherwise. In this case, the browser will redirect to the link address.